items.md

```diff
@@ -123,6 +122,22 @@ Administrators can directly create an archived item (bypassing the workflow). An
 }
 ```
 
+### Creating an archived item from an external source
+**POST /api/core/items**
```

Comment:
Thank you @benbosman for adding this. 
I hoped to find this solution in a different way. Having some distinguishable endpoint or parameter to allow this kind of process. 
What do you think if, to achieve the same result, we do it in two steps? First create an empty item, then, create metadata by linking it to the external source endpoint. 

Something like:

1. `POST /api/core/items?owningCollection=<:uuid>` to firstly create the new empty item
2. them updating metadata using a different from what is currently defined in the RestContract `Content-Type:text/uri-list`: `PUT /api/core/items/<:uuid>`. Like:

```
 curl -i -X PUT https://dspace7.4science.it/dspace-spring-rest/api/core/items/1c11f3f1-ba1f-4f36-908a-3f1ea9a557eb \
 -H "Content-Type:text/uri-list" \
 --data "https://dspace7.4science.it/dspace-spring-rest/api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436"
```

Reply:
Hi @paulo-graca 
If this becomes a two-step approach, we'll put an additional burden on the Angular UI. This would imply the button to import an external source in a new entity would require creating an empty item first, updating it with the external source hereafter, and continue work on the current item again. If you'd e.g. loose your network connection during this set of steps, you'll end up with an archived item without metadata.

What would be the problem with a single-step approach where the item is immediately created with the metadata populated?

Reply:
@benbosman and @paulo-graca : I'd prefer this be one step, though I wonder if the differentiator should simply be the content type.  In other words, you can send along either a `uri-list` or JSON to provide the metadata. But, either way, the endpoint could be `POST /api/core/items?owningCollection=<:uuid>`.

Would that make more sense?

We also could consider getting opinions here from the DSpace 7 team, as this need is not specific to Entities.

Reply:
I did check out the details from the uri-list again.
What I had found in https://tools.ietf.org/html/rfc2483#section-5 was that there was no need for all URIs to be of the same type.
On the other hand, when verifying samples, I haven't found a REST sample where multiple types of objects are passed in a URI-list.
For this reason, I've adjusted it based on Tim's recommendation